source/js/fellowships.js

```diff
@@ -0,0 +1,47 @@
+/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "React" }] */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import Person from './components/people/person.jsx';
+
+// Embed various Fellowships pages related React components
+function injectReactComponents() {
+  // Science Fellowship
+  if (document.getElementById(`featured-science-fellow`)) {
+    let metadata = {
```

Comment:
@alanmoo 

We could feed these hardcoded metadata via HTML `data-*` attributes... but some of them aren't string. It's more troublesome to hardcode array and object as `data-*` values... and it'll make the HTML code less readable. I can't find a DRYer way to do this... at least not now as these values are hardcoded.